plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -276,4 +308,17 @@ public void testCreateTableEmptyColumns()
         }
         return Optional.of(dataMappingTestSetup);
     }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
```

Comment:
nit: Move this to the top of the class.

Reply:
There are other enum values which you can set to false to prevent the commented tests (e.g. SUPPORTS_CREATE_SCHEMA).
I would suggest to take a look at the impl of the overridden empty tests and add more `SUPPORTS_*` in `hasBehaviour` accordingly. Once that is done it should be possible to remove all commented out tests.

Reply:
`io.trino.testing.AbstractTestDistributedQueries#testCreateSchema` doesn't have a `skipTestUnless(supportsCreateSchema())` in the beginning of the method.

This is why I've added an empty overridden method for `testCreateSchema()`

Reply:
I've moved now the `hasBehavior` method to the top of the class. Thank you for pointing this out.

Reply:
> io.trino.testing.AbstractTestDistributedQueries#testCreateSchema doesn't have a skipTestUnless(supportsCreateSchema()) in the beginning of the method.

that method has `if (!supportsCreateSchema()) {`

Reply:
I've modified `hasBehavior` method to reflect the fact that the schema creation is not supported.